userguide/tutorials/development.adoc

```diff
@@ -20,11 +20,16 @@ The first step is to clone the Kill Bill repository. For this, you need to follo
 
 . Click on *Clone a Git Repository and add the clone to this view*
 
-. In the Git Repository details, enter the Kill Bill Git URL https://github.com/killbill/killbill in the URI field. Enter your Git Credentials. Click *Next*
+. In the Git Repository details, enter the Kill Bill Git URL https://github.com/killbill/killbill in the *URI* field. 
+
+.. If you want read-write access to the Kill Bill repository, Select *https* in the *Protocol* dropdown and enter your Git credentials in the *User/Password* fields . 
```

Comment:
Note: read-write access for the `github.com/killbill/killbill` repository is restricted to the members of the core team. However, people can fork the code to their own account (e.g. to `github.com/learnjavawithreshma/killbill`), and this repository would be read-write. I would clarify this.

Reply:
Updated as mentioned above